Plugins/Masscan/server/plugin.js

```diff
@@ -0,0 +1,109 @@
+import { types, resolvers, loaders } from "./graphql";
+
+const settings = {
+  docker: {
+    name: "masscan",
+    dockerfile: `
```

Comment:
I'd love to figure out how to load this without copying/pasting as a string. We should consider doing things with the Meteor `Assets` library.